e2e/tests/init_test.go

```diff
@@ -105,6 +119,14 @@ type suite struct {
 	// moduleParamsMap is a map of module names to a map of parameter names to parameter values & types.
 	expectedModuleParams moduleParamsMap
 
+	// wsConn is the websocket connection to the PATH websockets endpoint.
+	wsConn *websocket.Conn
+	// numETHSubscriptionEvents is the number of eth subscription events received.
```

Comment:
Is my understanding here correct?

```suggestion
	// numETHSubscriptionEvents is the number of eth subscription events received
	// by all RelayMiners by all Gateways involved in the test.
```

Reply:
...events received from the RelayMiner through the Gateway's websocket connection.